design + content updates for brightwing reader

melaniebmn opened the pull request.

melaniebmn committed:
design + content updates for brightwing reader

coler-j commented on src/css/viewer.css:
```diff
@@ -428,7 +495,7 @@ html[data-theme=night-theme] #right-page-btn:hover
 
 @media only screen and (min-width: 768px) {
   #app-container.menu--show #reading-area {
-    width: calc(100% - 400px);
```
I think this may be required for viewport resizing.

melaniebmn replied:
Kaleeg wants the sidebar to be an overlay on top of the reading area instead of having the reading area size adjusted when the sidebar is open.

melaniebmn committed:
fix: save + apply setting changes

melaniebmn committed:
remove spacing around epub-reader-frame

melaniebmn committed:
open toc by default when sidebar is toggled

melaniebmn committed:
improve settings display on mobile

melaniebmn committed:
remove color options from settings + close menu on save

melaniebmn committed:
add preview text for setting changes

melaniebmn committed:
improve design of header

melaniebmn committed:
hide address bar on scroll (mobile)

melaniebmn committed:
improve mobile sidebar nav

melaniebmn committed:
add swipe functionality for mobile

melaniebmn committed:
remove unnecessary icons

melaniebmn committed:
update spinner animation

coler-j committed:
Minor fixes

- fixes screenfull.js by updating to v4.0.0
- removes tooltips from left/right page buttons (and from EpubReader.js)
- Add featureflag hide to download buttons

melaniebmn committed:
fixes night theme, active state sidebar menu, removed bg color for arrows

coler-j committed:
Integration of annotation overlay (WIP)

- Works on annotation addition
- Adds feature flags for tts and annotations
-

coler-j committed:
Merge branch 'mbockmann/brightwing-reader' of https://github.com/coler-j/readium-js-viewer into mbockmann/brightwing-reader
